### DISCUSSION TITLE
Code Signing

### DISCUSSION THREAD

**Discussion by compwiz21** [sircharlo/jw-launcher#130] [General]
Hey!
Brother Preston Barnes here, from Coalmont. I'd love to assist twith this project and offer to code sign your releases.
Also i just now realized that you developed this in electron and i have some questions. Give me a buzz. I sent you an email from deployitfast@gmail.com and my number is in it.  Give me a ring when you can!

Preston

**Comment by sircharlo** [comment]
Sorry for the delayed response! Thanks for the offer; and please let me know if you still have questions! 😄
